tox.ini

```diff
@@ -1,2 +1,2 @@
 [flake8]
-max-line-length = 120
+max-line-length = 150
```

Comment:
Screen sizes are different ;) and if one line breaks the 150 limit we will increase it to 160 :p
imho 120 is actually more than reasonable, also helps consistency over different repos.